slides/content/02-kotlin/_index.md

```diff
@@ -644,7 +641,7 @@ Kotlin classes have two types of members: **methods** and **properties**
 | Kotlin                 | **No** (Hidden)              | *Yes*   | *Yes*      |
 | C#                     | *Yes*                        | *Yes*   | *Yes*      |
 
-In Scala, at the caller site, methods and fields are hard to distinguish, as parentheses for 0-ary method calls are optional
```

Comment:
Now you cannot do:
```
def foo(): Unit = ()
foo
```
https://scastie.scala-lang.org/cric96/vPpqMslpSjeQxRWPw7Cmiw
So I think it is better to discuss only about the UAP